vault/changelog.release.2022-01-04.md

```diff
@@ -0,0 +1,115 @@
+---
+id: MQpi87qLnzKK5iK3FlxUp
+title: 0.76
+desc: ''
+updated: 1641321878137
+created: 1641314638791
+---
+
+Dendron 0.76 has sprouted  🌱
+
+Do you find yourself updating date values in the content of your inserted templates? That can now be inserted automatically with [[template variables|dendron://dendron.dendron-site/dendron.topic.templates#template-variables]]:
+
+- Example: `Today is {{ CURRENT_YEAR }}.{{ CURRENT_MONTH }}.{{ CURRENT_DAY }}`
+- Template output: `Today is 2022.01.04`
+
+Speaking of templates: we've added a new tutorial on [[Making Your First Schema|dendron://dendron.dendron-site/dendron.topic.schema.tutorial.first-schema]]. This week there will be a blog post version of the tutorial for sharing with the broader community. Take a look!
```

Comment:
its not clear how templates tie in to schemas with just this sentence. reword. no need to mention blog here.

> We've also added a new tutorial on [[Making Your First Schema|dendron://dendron.dendron-site/dendron.topic.schema.tutorial.first-schema]]. This walks you through creating a schema for your daily journal and automatically applying a template to it.